Makefile

```diff
@@ -0,0 +1,19 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+ifeq ($(WITH_COVERAGE), TRUE)
+	TEST_COMMAND = coverage run -m skbio/__init__
```

Comment:
This command fails locally for me:

```
$ WITH_COVERAGE=TRUE make test
coverage run -m skbio/__init__
No module named skbio/__init__
make: *** [test] Error 1
```

Changing it to `TEST_COMMAND = coverage run -m skbio.__init__` fixed it for me.